core/server/src/main/java/alluxio/worker/block/BlockWorker.java

```diff
@@ -629,4 +631,19 @@ public FileInfo getFileInfo(long fileId) throws IOException {
       throw new IOException(e);
     }
   }
+
+  /**
+   * Creates a file to represent a block denoted by the given block path. This file will be owned
+   * by the Alluxio worker but have 777 permissions so clients can read and write to the file
```

Comment:
Normally yes, but not with stickybit on the directory. I've updated the javadoc.